cookbooks/bcpc-hadoop/recipes/maven_config.rb

```diff
@@ -62,8 +62,10 @@
         -keystore #{node['bcpc']['hadoop']['java_https_keystore']} \
         -storepass changeit \
         -trustcacerts \
-        -import
```

Comment:
i think this should be retained

Reply:
there is already a '-import' 4 lines above